### PR TITLE
Deprecate old posts and change a few from conversion to awareness

### PR DESCRIPTION
**Pull request by mraible**

### CODE REVIEW

File: _source/_posts/2018-10-31-jwts-with-java.md [hunk 1]
```diff
@@ -11,7 +11,7 @@ tweets:
 - "Tutorial - JSON Web Tokens with @Java"
 - "Learn how to create JWTs with Java and #JJWT"
 image: blog/featured/okta-java-short-headphones.jpg
-type: conversion
+type: awareness
```
Comment (mraible) [inline _source/_posts/2018-10-31-jwts-with-java.md:14]: @bdemers Are you OK with this change? I believe you helped publish this one.

File: _source/_posts/2021-01-13-developers-cheatsheet-csharp-9.md [hunk 2]
```diff
@@ -11,7 +11,7 @@ tweets:
 - "Get up to speed on using #csharp9 and #csharp8 in this post"
 - "Wondering how C# 9 stacks up after C#8? We've got you covered with this cheat sheet!"
 image: blog/featured/okta-dotnet-books-quarter.jpg
-type: conversion
+type: awareness
```
Comment (mraible) [inline _source/_posts/2021-01-13-developers-cheatsheet-csharp-9.md:14]: @Quorralyne Are you OK with this change? I believe you helped publish this one.

File: _source/_posts/2021-03-05-ultimate-guide-to-password-hashing-in-okta.md [hunk 3]
```diff
@@ -11,7 +11,7 @@ tweets:
 - "Follow along with this interactive guide to learn how to hash passwords with Python"
 - "Use the live Python examples in this post to learn the details of hashing passwords!"
 image: blog/ultimate-guide-to-password-hashing-in-okta/header.png
-type: conversion
+type: awareness
```
Comment (mraible) [inline _source/_posts/2021-03-05-ultimate-guide-to-password-hashing-in-okta.md:14]: @jpf Are you OK with this change? I believe you helped publish this one.

File: _source/_posts/2018-10-31-jwts-with-java.md (hunk 1 above)
Comment (bdemers) [reply _source/_posts/2018-10-31-jwts-with-java.md:14]: 👍

**Review by bdemers — APPROVED** [review]

**Pull request merged**